modules/dreamview/frontend/src/components/StatusBar/Electricity.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+
+@observer
+export default class Electricity extends React.Component {
+
+  render() {
+    const {
+      electricityPercentage,
+    } = this.props;
+
+    const percentageString = (electricityPercentage !== undefined) ? `${electricityPercentage}%` : 'UNKNOWN';
+    const rectWidth = 24 * (electricityPercentage / 100);
+    return (
+            <div className="show-electricity">
+                <div className="battery-icon">
+                    <svg version="1.1">
+                        <rect rx="2" ry="2" x="44" y="10" width="30" height="15" fill="rgb(15,127,18)"/>
+                        <rect x="46" y="12" width="26" height="11" fill="rgb(0,0,0,0.5)"/>
+                        <rect x="47" y="13" width={ rectWidth } height="9" fill="rgb(15,127,18)"/>
+                        <rect rx="2" ry="2" x="74" y="13" height="8" width="2" fill="rgb(15,127,18)"/>
+                    </svg>
```

Comment:
as we are using svg to show battery label, do we want to change it to red when percentage is low?